scripts/docs_update_release.py

```diff
@@ -0,0 +1,224 @@
+from os import sep, path, environ
+import json
+import re
+
+from datetime import date
+
+RELEASES_JSON_PATH = f".{sep}src{sep}Resources{sep}download{sep}releases.json"
+INDEX_PATH = f".{sep}src{sep}Resources{sep}download{sep}index.html"
+INNO_SETUP_PATH = f".{sep}src{sep}InnoSetup{sep}IdfToolsSetup.iss"
```

Comment:
Do we need to use 'sep' here? I believe this won't be a multi-platform app and using the separator character would make it more readable IMO